base/bucket_gocb.go

```diff
@@ -23,11 +23,12 @@ import (
 	"strconv"
 	"strings"
 
+	"sync"
```

Comment:
The empty line above sync should be removed (I realize sync was already misplaced in the imports before this PR).